test_framework.js

```diff
@@ -0,0 +1,16 @@
+const assertEquals = (expected, actual) => expected === actual;
+const assertNotEquals = (expected, actual) => expected !== actual;
+const assertTrue = actual => actual === true;
+const assertFalse = actual => actual=== false;
+const assertTypeOf = (expected, actual) => expected === typeof actual;
+const assertThrows = actual => typeof actual === typeof new Error();
```

Comment:
Approves any error (Example: reference error) that gets thrown. Not specifically a manual user-thrown error.
- Can be fixed by using a more specifically coded assert function that checks for the type of error an the error message as well.